samples/createSource.ts

```diff
@@ -1,7 +1,7 @@
-import {Source, SourceVisibility} from '@coveo/push-api-client';
+import {PushSource, SourceVisibility} from '@coveo/push-api-client';
```

Comment:
Similar to my other comment, another approach is to leave the old code (and name) and return the new implementation instead, internally.

And mark the `Source` as deprecated + warning, so that people have time to upgrade.

Granted, this client is not the most used out there at Coveo, so we can be more liberal with breaking changes.

But... just so you know, this is typically the way it can be done.